app/views/equipment/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>Equipment List</h1>
+<h1>Offers List</h1>
```

Comment:
I think keep it as "Equipment" instead of "Offers" sounds more user friendly~

Reply:
I have no preference. Equipment sounds more technical, offers sounds more marketplace

Reply:
Sorry I merged before I saw your comment @tieuu